### DISCUSSION TITLE
Terms of Use

### DISCUSSION THREAD

**Discussion by Artistblock307** [nesbox/TIC-80#1403] [General]
I would rather not be, "that guy" but I want to know what is TIC-80's stance on this game? https://tic80.com/play?cart=1006 Does it break the Terms of Use?

**Comment by joshgoebel** [comment]
Could you be clearer which Terms of Use criterion you feel this might possibly be in violation of?  I'd hate to be "that guy" and assume... :)

**Reply by Artistblock307** [reply]
"Do not post material that violates copyright, obscenity or any other laws."
I mean specifically obscenity thing. If I understand the law the internet is mostly closed off for children under the age of 13. But I don't know if this is appropriate for anyone under the age of 18 because of the suggestive nature. A NSFW warning might be a good idea?

**Reply by Artistblock307** [reply]
I mentioned this in the discord and they correctly informed me that the woman is fully naked in the sprite editor.

**Comment by Artistblock307** [comment]
To answer your question...Yes. Twice.

**Reply by joshgoebel** [reply]
When I took a glance the game play was censored.  Guess @nesbox will decide what to do here.

**Comment by nesbox** [comment]
I don't want to remove this content from the website, as a variant we could mark it as NSFW and allow people to decide in the profile settings they want to see such content.
For example, itch.io has `Show content marked as adult in search & browse` checkbox in the settings, we could do the same...

**Comment by sssilver** [comment]
Casting my vote here that the content should stay. I'm happy it exists; and if we need censorship, we should strive to bring it to a minimum.

**Comment by tjpalmer** [comment]
Some schools for minors use TIC-80. Worth being cautious.
